crates/sui-core/src/unit_tests/authority_tests.rs

```diff
@@ -5129,157 +5134,12 @@ async fn test_consensus_commit_prologue_generation() {
             .next()
             .unwrap()
     };
+    // Compare clock version between the prologue (index 0) and the transaction that uses clock (index 2)
     let clock_v1 = get_assigned_version(&processed_consensus_transactions[0].key());
-    let clock_v2 = get_assigned_version(&processed_consensus_transactions[1].key());
+    let clock_v2 = get_assigned_version(&processed_consensus_transactions[2].key());
     assert!(clock_v1 < clock_v2);
 }
 
-#[tokio::test]
-async fn test_consensus_message_processed() {
```

Comment:
confirming that deleting this test is intentional?

Reply:
yes, I don't think its necessary - so many things would break if deduplication stopped working